packages/@okta/vuepress-theme-prose/components/OnThisPageItem.vue

```diff
@@ -17,31 +17,46 @@ export default {
   data() {
     return {
       imActive: false,
-      iHaveChildrenActive: false
+      iHaveChildrenActive: false,
+      linkHash: '',
     }
   },
   components: {
     OnThisPageItem: () => import('../components/OnThisPageItem.vue'),
   },
   mounted() {
     this.setActiveData();
+    this.calculateLinkHash();
```

Comment:
does this need to happen on update as well?
also a nit-pick: `calculate` + `hash` reads like a checksum related operation - perhaps we could use a different name